dashboard/src/recoil/comments.js

```diff
@@ -11,17 +14,32 @@ export const commentsState = atom({
 const encryptedFields = ['comment', 'person', 'action', 'group', 'team', 'user', 'date', 'urgent'];
 
 export const prepareCommentForEncryption = (comment) => {
-  const decrypted = {};
-  for (let field of encryptedFields) {
-    decrypted[field] = comment[field];
-  }
-  return {
-    _id: comment._id,
-    createdAt: comment.createdAt,
-    updatedAt: comment.updatedAt,
-    organisation: comment.organisation,
+  try {
+    const decrypted = {};
+    if (!looseUuidRegex.test(comment.person) || !looseUuidRegex.test(comment.action)) {
+      throw new Error('Comment is missing person or action');
+    }
+    if (!looseUuidRegex.test(comment.team)) {
+      throw new Error('Comment is missing team');
+    }
+    if (!looseUuidRegex.test(comment.user)) {
+      throw new Error('Comment is missing user');
+    }
+    for (let field of encryptedFields) {
+      decrypted[field] = comment[field];
+    }
+    return {
+      _id: comment._id,
+      createdAt: comment.createdAt,
+      updatedAt: comment.updatedAt,
+      organisation: comment.organisation,
 
-    decrypted,
-    entityKey: comment.entityKey,
-  };
+      decrypted,
+      entityKey: comment.entityKey,
+    };
+  } catch (error) {
+    toast.error("Désolé, une erreur technique est survenue, l'équipe technique a été prévenue.");
```

Comment:
Et si on commençait à être plus précis ?

```suggestion
    toast.error("Le commentaire n'a pas été sauvegardé car son format était incorrect. Vous pouvez vérifier son contenu et tenter de le sauvegarder à nouveau. L'équipe technique a été prévenue et va travailler sur un correctif.");
```